src/runtime/model/animated.rs

```diff
@@ -65,11 +65,12 @@ impl Transform {
         let skew_matrix = if skew != 0.0 {
             const SKEW_LIMIT: f64 = 85.0;
             let skew = -skew.min(SKEW_LIMIT).max(-SKEW_LIMIT);
+            let skew = skew.to_radians();
             let angle = skew_angle.to_radians();
             fn make_skew(x: f64) -> Affine {
-                Affine::new([1.0, x.tan(), 0.0, 1.0, 0.0, 0.0])
+                Affine::new([1.0, 0.0, (x).tan(), 1.0, 0.0, 0.0])
```

Comment:
I think it was originally wrong, btw.
See this commit that "fixed" the skew logic: https://github.com/loopystudios/vellottie/commit/aaab5c52bf3862cf1b6348813ab390c458b7d29e

I just used `Affine::skew` at least, that works.

Was there a better implementation you had in mind?

I also have no idea what that SKEW_LIMIT you wrote in does. It seems when the skew is over that limit I get bad results, too. Here's an example file with 120 skew that looks *too skewed* in velato:

[lottie(2).json](https://github.com/linebender/velato/files/14711921/lottie.2.json)

Reply:
Just using `Affine::skew` was all I intended here. I took the skew limit from Skottie but it might be wrong. It’s worth looking at other implementations but no need to block this PR on that.